lib/src/builder/parsing/members/factory.dart

```diff
@@ -87,7 +87,7 @@ class BoilerplateFactory extends BoilerplateMember {
 
     final names = FactoryNames(factoryName);
     final generatedFactoryName = names.implName;
-    final generatedConfigName = names.configName;
+    final generatedConfigName = names.privateConfigName;
 
     final initializer = variable.initializer;
     final referencesGeneratedFactory = initializer != null &&
```

Comment:
For backward compatibility, this should be looking for either the public or the private version, and not just the private one.

This logic is duplicated by `.usesAGeneratedConfig`, so we should just use that instead.

I would expect this to cause validation to fail on the public config... is it not? It looks like there's test coverage around it, but for some reason it's not failing... 😕

Reply:
@greglittlefield-wf `generatedConfigName` is only used in the [error message](https://github.com/Workiva/over_react/blob/8ce88addddd95308d85403900d3dae3e8e4cb8e2/lib/src/builder/parsing/members/factory.dart#L98-L107) which is why the build isn't failing. I was thinking it was okay to just use the private config name here since the error message is showing examples of what should be done, but I can add the public config name to the error message if you think it's needed!

Reply:
Oh wow, I read this wrong, and thought this code:
```dart
    final referencesGeneratedFactory = initializer != null &&
        anyDescendantIdentifiers(
            initializer, (identifier) => identifier.name == generatedFactoryName);
```
was referencing `generatedConfigName`, but it's referencing `generatedFactoryName` instead. My bad!

No, the private config is good here. Sorry for the confusion!

Reply:
Oh I see! No worries!